src/components/02-elements/typography/typography.scss

```diff
@@ -173,5 +173,10 @@ p {
   }
 }
 
+p > a {
+  text-decoration: underline;
+}
 
-
+ul > li > a {
```

Comment:
Not sure the `ul` is necessary here as a `li` is just a descendent of `ul` anyway